doc/Type/Metamodel/TypePretense.pod6

```diff
@@ -0,0 +1,75 @@
+=begin pod :kind<Type> :subkind<class> :category<metamodel>
+
+=TITLE role Metamodel::TypePretense
+
+=SUBTITLE Metarole for type pretenses
+
+    role Metamodel::TypePretense { }
+
+I<Warning>: this role is part of the Rakudo implementation, and is not
+a part of the language specification.
+
+Roles can type-check as C<Mu>, C<Any>, and C<Cool>, but don't actually
```

Comment:
Not clear if you mean that all roles do that, using this mechanism. Maybe you could clarify.

Reply:
All roles do. "Any role will" would probably be clearer than "Roles can" here